src/frontend/js/components/DashboardOrderLoader/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useParams } from 'react-router-dom';
+import { useBreadcrumbsPlaceholders } from 'hooks/useBreadcrumbsPlaceholders';
+import { useOrder } from 'hooks/useOrders';
+import { useProduct } from 'hooks/useProduct';
+import { Spinner } from '../Spinner';
+import Banner, { BannerType } from '../Banner';
+import { DashboardItemOrder } from '../DashboardItem/Order/DashboardItemOrder';
+
+export const DashboardOrderLoader = () => {
+  const params = useParams<{ orderId: string }>();
+  const {
+    states: { error, fetching },
+    ...order
+  } = useOrder(params.orderId);
+
+  const product = useProduct(order.item?.product);
```

Comment:
This hook should be enabled only once order is defined I think.

Reply:
It is done automatically inside `useResource`, see here: https://github.com/openfun/richie/blob/617353438f17660f783f73fffc89a129381d9086/src/frontend/js/hooks/useResources/index.tsx#L71

Reply:
Ok we'll do it in a separate PR